viewer/src/public/migration/Cognite3DModel.ts

```diff
@@ -209,48 +221,72 @@ export class Cognite3DModel extends THREE.Object3D implements CogniteModelBase {
 
   async setNodeColor(nodeId: number, r: number, g: number, b: number): Promise<void> {
```

Comment:
I still think this sounds too much like the old api's set color, where you changed the color of the node. If I am not mistaken this sets the color of the node if it isn't highlighted currently.

Last thing I want is people asking us why the color doesn't change when you try and set the color, and we figure out that they already have the node selected or filtered or who knows what effect we have on it which overrides the color.
Suggestions:
`setBaseColor`
`setDefaultColor` <-- don't really like this one.
`setStandardColor`

Reply:
I agree, we should aim for people providing `NodeStyle` instead to define all styling aspects. For now however I think keeping it close to the old API makes sense.